lms/resources/_js_config/__init__.py

```diff
@@ -512,6 +523,18 @@ def _hypothesis_client(self):
             ]
         }
 
+    def forward_lti_parameters(self):
+        form_fields = {
+            param: value
+            for param, value in self._request.lti_params.items()
+            # Don't send over auth related params. We'll use our own
+            # authorization header
+            if param
+            not in ["oauth_nonce", "oauth_timestamp", "oauth_signature", "id_token"]
+        }
+        form_fields["authorization"] = self._auth_token()
+        return form_fields
```

Comment:
I reckon we should eventually (soon-ish) re-sign this to forbid any tampering.

I do think we should do that for all endpoints were we forward information to ourselves so we could handle that separately as a general mechanism.